### DISCUSSION TITLE
Some more information about retention policy

### DISCUSSION THREAD

**Discussion by trexxeon** [garethgeorge/backrest#170] [Ideas]
It would be nice to have a bit of more explanation for the retention policy "by time period"

I guess this relates to the documentation for restic removing snapshots according to policy? that "hourly 24" means keep one snapshot of each hour for 24 hours? maybe a popup text with info for each field? e.g. "Monthly 3: keep the most recent monthly snapshot for 3 months"

**Comment by garethgeorge** [comment]
Hey, all of backrest's options map directly to the flags on forget: https://restic.readthedocs.io/en/stable/060_forget.html#removing-backup-snapshots.
